File: VB/T501713/Root.Master.vb
Imports DevExpress.Web
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.UI
Imports System.Web.UI.WebControls

Namespace T501713
    Partial Public Class Root
        Inherits System.Web.UI.MasterPage

        Protected Sub Page_Load(ByVal sender As Object, ByVal e As EventArgs)

        End Sub

        Public Event PopupInit As EventHandler
        Protected Sub popupControl_Init(ByVal sender As Object, ByVal e As EventArgs)
            RaiseEvent PopupInit(sender, e)
            popupControl.FooterText = "Rendered by the master page"
            popupControl.ShowFooter = True
        End Sub

        Public Event PopupCallback As PopupWindowCallbackEventHandler
        Protected Sub popupControl_WindowCallback(ByVal source As Object, ByVal e As PopupWindowCallbackArgs)
            RaiseEvent PopupCallback(source, e)
        End Sub

    End Class
End Namespace

File: CS/T501713/Root.Master.cs
using DevExpress.Web;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.UI;
using System.Web.UI.WebControls;

namespace T501713 {
    public partial class Root : System.Web.UI.MasterPage {
        protected void Page_Load(object sender, EventArgs e) {

        }

        public event EventHandler PopupInit;
        protected void popupControl_Init(object sender, EventArgs e) {
            if (PopupInit != null)
                PopupInit(sender, e);
            popupControl.FooterText = "Rendered by the master page";
            popupControl.ShowFooter = true;
        }

        public event PopupWindowCallbackEventHandler PopupCallback;
        protected void popupControl_WindowCallback(object source, PopupWindowCallbackArgs e) {
            if (PopupCallback != null)
                PopupCallback(source, e);
        }

    }
}